### PR TITLE
feat (dependencies): Add Laravel 6 support

### PR DESCRIPTION
**Pull request by pablo-bossi**
Add support for Laravel 6.
Needed only to update dependencies.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,13 +6,13 @@
         }
     },
     "require": {
-        "laravel/framework": "~5.1"
+        "laravel/framework": "~6.0"
```
Comment (pablo-bossi) [inline composer.json:9]: Support for Laravel 6 (Given dependencies is either 5 or 6)

File: composer.json [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*",
-        "orchestra/testbench": "~3.1",
-        "phpunit/phpunit": "~4.7",
-        "satooshi/php-coveralls": "dev-master@dev"
+        "mockery/mockery": "1.3.*",
+        "orchestra/testbench": "4.*",
+        "phpunit/phpunit": "~8.0",
+        "php-coveralls/php-coveralls": "2.2.*"
```
Comment (pablo-bossi) [inline composer.json:15]: satooshi/php-coveralls is deprectated so replaced by the suggested replacement

File: composer.json [hunk 3]
```diff
@@ -6,13 +6,13 @@
...
+        "laravel/framework": "~6.0"
     },
     "require-dev": {
-        "mockery/mockery": "0.9.*",
-        "orchestra/testbench": "~3.1",
-        "phpunit/phpunit": "~4.7",
-        "satooshi/php-coveralls": "dev-master@dev"
+        "mockery/mockery": "1.3.*",
+        "orchestra/testbench": "4.*",
+        "phpunit/phpunit": "~8.0",
```
Comment (pablo-bossi) [inline composer.json:14]: Updated dependencies to match Laravel 6 needs

File: phpunit.xml [hunk 4]
```diff
@@ -7,8 +7,7 @@
      convertNoticesToExceptions="true"
      convertWarningsToExceptions="true"
      processIsolation="false"
-     stopOnFailure="false"
-     syntaxCheck="false">
```
Comment (pablo-bossi) [inline phpunit.xml:11]: syntaxCheck is not recongized on phpUnit 8

**Review by pablo-bossi — COMMENTED** [review]
The package is in use on a local project with this updates

**Comment by esbenp** [comment]
Thanks @pablo-bossi !

**Pull request merged**